aidb/inference/examples/detectron_local_ocr.py

```diff
@@ -5,12 +5,18 @@
 """
 import numpy as np
 import pandas as pd
-import pdf2image
 
-import pytesseract
-from detectron2 import model_zoo
-from detectron2.engine import DefaultPredictor
-from detectron2.config import get_cfg
+try:
+  import pdf2image
+except ImportError:
```

Comment:
I think we should error and tell the user to install instead of installing on their behalf